beetsplug/embedart.py

```diff
@@ -107,13 +110,39 @@ def embed_func(lib, opts, args):
                     art.embed_item(self._log, item, imagepath, maxwidth,
                                    None, compare_threshold, ifempty,
                                    quality=quality)
+            elif opts.url:
+                try:
+                    response = requests.get(opts.url, timeout=5)
+                    response.raise_for_status()
+                except requests.exceptions.RequestException as e:
+                    self._log.error("Error: {}".format(e))
```

Comment:
I think you could even remove the "Error: " from the string. It's not necessary IMHO and would even reduce the length of an possibly already lengthy error message that (sometimes) might include the word error: 

E.g

```
embedart: Error: 400 Client Error: Bad Request for url: https://geo-media.beatport.com/image_size/1400x1400/d7c35661-1dfb-49c2-92cf-056352615872.jpX
```

```
embedart: Error: No connection adapters were found for 'ftp://geo-media.beatport.com/image_size/1400x1400/d7c35661-1dfb-49c2-92cf-056352615872.jpg'
```